packages/client/src/HistoryApi.ts

```diff
@@ -0,0 +1,26 @@
+import { PartitionsResponse } from "@lionweb/repository-common";
+import { ClientResponse, RepositoryClient } from "./RepositoryClient.js";
+import { RetrieveResponse } from "./responses.js";
+
+export class HistoryApi {
```

Comment:
What happens if I call the HistoryApi after having instantiated the database without support of history?
And vice-versa?

Should we specify it in the documentation of the class?

Reply:
You will get a response with success === false (as with any unsuicessful LionWeb call.
I added a description stating this.